src/copy-button/copy-button.ts

```diff
@@ -0,0 +1,35 @@
+import { bindable } from 'aurelia-framework';
+
+export class CopyButton {
+  @bindable public targetId: string;
+  public showCheckIcon: boolean = false;
+  
+  public copy() {
+    var url = document.getElementById(this.targetId);
```

Comment:
ik twijfel of het beter zou zijn om het html element zelf te binde ipv de id; zo hoeft niet elk element een hard id te hebben.